paasta_tools/secret_providers/__init__.py

```diff
@@ -49,4 +50,11 @@ def get_secret_signature_from_data(self, data: Mapping[str, Any]) -> Optional[st
 
 
 class SecretProvider(BaseSecretProvider):
-    pass
+    # from .vault import CryptoKey # cannnot import of cyclical dependency
+    def get_key_versions(
+        self, key_name: str, mount_point: str = "keystore"
```

Comment:
are we planning on ever using this mountpoint arg? it feels a little weird to have this be in the interface given that this is so vault-specific

Reply:
Mountpoint is set on Vault backend so as long as it doesn't change, then we can hard code mountpoint. So it shouldn't be used mostly.